ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -134,7 +135,55 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
     public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (url.startsWith("http://") || url.startsWith("https://") ||
             url.startsWith("file://")) {
-          return false;
+              boolean shouldBlock = false;
```

Comment:
Looking over this again, I don't think this will solve the issue my PR is attempting to solve. The problem there (#10055) is with custom URL schemes. If you can only block http/https/file requests I mean.